### DISCUSSION TITLE
Auto-complete Input

### DISCUSSION THREAD

**Discussion by phoskee** [zauberzeug/nicegui#819] [Ideas / Feature Requests]
Hi guys!! Bad idea machine here.
The question is: leaving aside ui.select, is there an auto-complete input?
If the answer is yes, well, another bad idea.
If the answer is no, WELL, taa dam!!! Here it is:

![Animation](https://user-images.githubusercontent.com/125825348/233855765-63db7620-6a0c-449b-aee8-1d8fc5ab68d8.gif)

(okay, maybe it not looks like an auto-complete, but I think it gives the idea)

**Comment by rodja** [comment]
Interesting. We have not jet any auto-completion capability for ui.input. I'm curious how you implemented it. Do you have a pull request ready?

**Reply by phoskee** [reply]
Hi @rodja ! 
A simple but very rudimentary information exchange between a list, a for loop, and the shadow-text.
I would like to create a pr, but I would like to test it on input.py first, and I don't know how to do that. Give me very little time and I will try. If you want I can share the resulting code from the tests I did.

**Reply by rodja** [reply]
Sounds good

**Reply by phoskee** [reply]
I already have an open PR
